virtcontainers/kata_agent.go

```diff
@@ -59,7 +59,6 @@ var (
 // KataAgentConfig is a structure storing information needed
 // to reach the Kata Containers agent.
 type KataAgentConfig struct {
-	GRPCSocket   string
```

Comment:
Yes I agree, this info will never come from the configuration, and then there is no reason to store it in the config structure. State structure is the one where we store runtime values.